common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -228,6 +246,10 @@ private boolean isPkeyAuthUrl(@NonNull final String url) {
         return url.startsWith(AuthenticationConstants.Broker.PKEYAUTH_REDIRECT.toLowerCase(Locale.ROOT));
     }
 
+    private boolean isPasskeyUrl(@NonNull final String url) {
+        return url.startsWith(FidoConstants.PASSKEY_PROTOCOL_REDIRECT.toLowerCase(Locale.ROOT));
```

Comment:
You mentioned the registration flow, would it use the same uri format?

(I assume we should never hit the scenario where old common get a registration uri - as that should only be triggered when the client says that it can handle registration).

Reply:
It most likely would, though I would ask the server team to update the version parameter to "2.0". With the current logic, if the version is anything other than "1.0", then we will send back an exception that tells them we don't support the protocol version they are using.